Library/Homebrew/utils/github.rb

```diff
@@ -699,4 +699,14 @@ def multiple_short_commits_exist?(user, repo, commit)
 
     output[/^Status: (200)/, 1] != "200"
   end
+
+  def repo_commit_count_for_user(nwo, user)
+    return if Homebrew::EnvConfig.no_github_api?
+
+    commits = 0
+    API.paginate_rest("#{API_URL}/repos/#{nwo}/commits", query: "&author=#{user}") do |result|
```

Comment:
I feel like `#paginate_rest` should handle adding the `&` where necessary, instead of leaving it to callers? (Unless I'm misunderstanding how this works)